internal/web/web_test.go

```diff
@@ -78,3 +80,52 @@ func TestDuration(t *testing.T) {
 		})
 	}
 }
+
+func TestHTMLResponseError_ResponseBody(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name            string
+		err             error
+		wantBody        []byte
+		wantContentType string
+		wantErr         bool
+	}{
+		{
+			name:            "with error",
+			err:             errors.New("<error>"),
+			wantBody:        []byte(`<p class="error">&lt;error&gt;</p>`),
+			wantContentType: "text/plain",
+			wantErr:         false,
+		},
+		{
+			name:            "nil error",
+			err:             nil,
+			wantBody:        nil,
+			wantContentType: "",
+			wantErr:         true,
+		},
+		{
+			name:            "empty error",
+			err:             errors.New(""),
+			wantBody:        []byte(`<p class="error"></p>`),
+			wantContentType: "text/plain",
+			wantErr:         false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			e := NewHTMLResponseError(tt.err)
+			body, contentType, err := e.ResponseBody()
+			if tt.wantErr {
+				assert.Error(t, err)
```

Comment:
Change these to require.Error

Reply:
Require will error immediately while assert will fail the test but keep executing.